tests/fixtures/png-matcher.ts

```diff
@@ -0,0 +1,101 @@
+import { expect, type MatcherResult } from "bun:test"
```

Comment:
pull main and use the existing (and better) implementation with antialiasing handling